### DISCUSSION TITLE
What is Minio for?

### DISCUSSION THREAD

**Discussion by aasmundkamphaug** [Security-Onion-Solutions/securityonion#11860] [Unsupported Versions] [answered]
Hi, i noticed that that it was states files for Minio. I was wondering what minio is for, since i can not find it in the docs

**Comment by TOoSmOotH** [comment, marked as answer]
This was an experimental feature we never fully implemented. It was a possible replacement for redis but the s3 input plugin for logstash did not load balance properly with multiple nodes.
